### ISSUE TITLE
Error in plugin: failed to lint

### ISSUE THREAD

**Issue by joetm** [karimsa/gulp-jslint#28]
Gulp-jslint gives me some results. But it aborts with an error:

    Error in plugin 'gulp-jslint'
    Message:
        failed to lint <file>

So the plugin does not seem to be stable or compatible with any JavaScript code.

**Comment by karimsa** [comment]
Are linters not meant to throw errors when they encounter bad code...

**Comment by joetm** [comment]
Well, it's supposed to alert me. Not just stop and quit execution entirely.

**Comment by karimsa** [comment]
Well it is intended to stop the current build, because usually one does not wish to continue to do other tasks if a lint fails.

**Comment by joetm** [comment]
Okay, makes sense. However, jslint is different in many ways. 99% of the "errors" that I get in jslint do not produce any errors in my application. It's mostly just cosmetics and conventions. It very rarely happened to me that I encountered an actual error with jslint.

**Comment by karimsa** [comment]
You can try one of two things in that case:

1) if there's thing you don't want linted, such as whitespacing (which I'm assuming you meant by cosmetics), add the proper directives in `.jslintrc`:

```
{
  "white": true
}
```

2) Check out [gulp-plumber](https://github.com/floatdrop/gulp-plumber). I just popped it into a gulpfile with gulp-jslint and tried to lint a project full of errors, and it at least continues the build rather than stopping it, but the output is very ugly. For the output, you could just implement a custom gulp-jslint reporter.

**Comment by joetm** [comment]
I have added plumber recently, too.
My problem was that your module had no output at all. So I know there was something in my JavaScript file that caused the linting to abort, but I do not what.

**Comment by WazzaJB** [comment]
This is still an issue in my opinion and should be reopened, I have this running in a validation watch task and it kills execution. Would it be worth looking at adding an option for soft errors? Or something of the sort?

Thanks for the work on the linter integration with gulp, very helpful indeed :)
